packages/doc/src/components/Navigation/Navigation.jsx

```diff
@@ -109,23 +96,44 @@ const StyledListItem = styled.li`
   `}
 `;
 
-const Colapsible = styled.div`
+const NavigationLabel = styled.button`
   ${({
+    level,
     theme: {
       yoga: {
-        colors: { text },
+        colors: { text, primary },
       },
     },
   }) => `
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 0;
+  font-family: inherit;
+  letter-spacing: 0.5px;
+  line-height: 2;
+  font-size: 0.9rem;
+  text-decoration: none;
+  font-weight: 300;
+  align-items: center;
+  width: 100%;
+  background: none;
+  border: none;
+  transition: all 200ms ease-out;
+  color: ${hexToRgb(text.secondary, 0.75)};
+  text-indent: calc(15px * ${level});
+  outline-offset: 2px;
```

Comment:
Can you move all CSS properties which doesn't make use of the callback function outside of it?